api/src/main/java/org/openmrs/module/referencemetadata/RolePrivilegeMetadata.java

```diff
@@ -144,16 +148,97 @@ public void install() throws Exception {
         install(role(_Role.APPLICATION_HAS_SUPERUSER_PRIVILEGES, "Extends the underlying System Developer API role",
                 idSet(RoleConstants.SUPERUSER), idSet()));
 
-        install(role(_Role.APPLICATION_APPOINTMENT_VIEWER,
-                "Gives user the ability to view appointment schedules (but not to modify them)", idSet(), idSet(
-                _Privilege.APP_APPOINTMENTSCHEDULINGUI_HOME,
-                "App: appointmentschedulingui.viewAppointments")));
-
         install(role(_Role.APPLICATION_EDITS_EXISTING_ENCOUNTERS,
                 "Gives user the ability to edit patient encounter", idSet(), idSet(
                         _Privilege.TASK_EMR_PATIENT_ENCOUNTER_DELETE,
                         _Privilege.TASK_EMR_PATIENT_ENCOUNTER_EDIT)));
 
+        Module appointmentscheduling = ModuleFactory.getModuleById("appointmentscheduling");
```

Comment:
Don't you think a name like appointmentschedulingModule would make the code clearer?

Reply:
> Don't you think a name like appointmentschedulingModule would make the code clearer?

Yes, it makes sense.